model/jpa/src/test/resources/META-INF/persistence.xml

```diff
@@ -40,6 +40,10 @@
             <property name="hibernate.show_sql" value="false"/>
             <property name="hibernate.format_sql" value="false"/>
             <property name="hibernate.transaction.flush_before_completion" value="true"/>
+            <property name="hibernate.cache.use_second_level_cache" value="true"/>
+            <property name="hibernate.cache.use_query_cache" value="true"/>
+            <property name="hibernate.cache.region.factory_class" value="org.hibernate.cache.infinispan.JndiInfinispanRegionFactory" />
+            <property name="hibernate.cache.infinispan.cachemanager" value="java:CacheManager/entity"/>
```

Comment:
Does this work in "standalone" profile too ?
The GSOC branch (kafka) is able to run on that profile, and it would be great to keep that spirit :-)

Reply:
It should, but I will check to be sure.

Reply:
this is only relevant for the GSoC branch ;-) since master/1.1.x-dev do require the `-full` profile, due to JMS